lib/cylc/command_polling.py

```diff
@@ -1,99 +0,0 @@
-#!/usr/bin/env python2
-
-# THIS FILE IS PART OF THE CYLC SUITE ENGINE.
-# Copyright (C) 2008-2019 NIWA & British Crown (Met Office) & Contributors.
-#
-# This program is free software: you can redistribute it and/or modify
-# it under the terms of the GNU General Public License as published by
-# the Free Software Foundation, either version 3 of the License, or
-# (at your option) any later version.
-#
-# This program is distributed in the hope that it will be useful,
-# but WITHOUT ANY WARRANTY; without even the implied warranty of
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-# GNU General Public License for more details.
-#
-# You should have received a copy of the GNU General Public License
-# along with this program.  If not, see <http://www.gnu.org/licenses/>.
-"""Encapsulates polling activity for CLI commands."""
```

Comment:
This module is still used. Travis CI failures confirm.

Reply:
It's used in `cylc stop` and `cylc suite-state`.  We probably won't need it in the former anymore thanks to duplex comms, but probably do still need it in the latter (DB not server).